lib/dependencies.js

```diff
@@ -40,5 +45,6 @@ container.provide("moment", require("moment"));
 container.provide("proxywrap", require("findhit-proxywrap"));
 container.provide("restify", require("restify"));
 container.provide("restifyLinks", require("restify-links"));
+container.provide("tfa", require("2fa"));
```

Comment:
This is because we can't the namespace of an object start with a number.  I renamed it to twofa instead to keep the sounding the same as the library.